test/e2e/quadlet/idmapping-override.container

```diff
@@ -0,0 +1,13 @@
+## assert-podman-args "--uidmap=0:10000:10"
+## assert-podman-args "--uidmap=10:20000:10"
+## assert-podman-args "--gidmap=0:10000:10"
+## assert-podman-args "--gidmap=10:20000:10"
+## !assert-podman-args --userns=auto
+
+[Container]
+Image=localhost/imagename
+UIDMap=0:10000:10
+UIDMap=10:20000:10
+GIDMap=0:10000:10
+GIDMap=10:20000:10
+RemapUsers=auto
```

Comment:
I prefer throwing an error over (silently) ignoring user-specified options/data.  Much easier to debug.

Reply:
The `Remap` keys are deprecated and are not even documented. The only reason the code still looks at them is for backward compatibility.
The current behavior was that if `UserNS` is set, they are disregarded. So, I just did the same

Reply:
Fine with me, but can we at least generate a warning.

Reply:
Currently, we're not generating any warning. Is it really worth it for deprecated and undocumented keys?

Reply:
If you were a user and did not know the key was deprecated how would you find out?

Reply:
I understand the desire. But, cost benefit wise I'm not sure it's worth it.

Currently, logging is handled in [main.go](https://github.com/containers/podman/blob/main/cmd/quadlet/main.go) in case of errors, while the actual processing is done in the [pkg](https://github.com/containers/podman/blob/main/pkg/systemd/quadlet/quadlet.go).

I don't think moving the logging into the pkg is correct but on the other hand the pkg does not have a means to emit warnings to the main code (only errors).

How about adding a comment about it in the man page?

Reply:
I'd really just return an error.  Suppressing/ignoring (or even only logging) is debugging time for users.

Reply:
OK. So, I kept the support for the deprecated keys without any deprecation notice. However, if the user sets any of the deprecated keys along with any of the new explicit keys Quadlet will return an error.
I hope this is OK.